test/test_bbm_bbm_1d.jl

```diff
@@ -11,15 +11,15 @@ EXAMPLES_DIR = joinpath(examples_dir(), "bbm_bbm_1d")
     @trixi_testset "bbm_bbm_1d_basic" begin
         @test_trixi_include(joinpath(EXAMPLES_DIR, "bbm_bbm_1d_basic.jl"),
                             tspan=(0.0, 1.0),
-                            l2=[0.0032744047423353103 0.0072467844128150435],
-                            linf=[0.0027330692683324997 0.0037734832582039246],
-                            cons_error=[3.61905099297374e-14 5.684341886080802e-14],
-                            change_waterheight=3.61905099297374e-14,
-                            change_velocity=-5.684341886080802e-14,
-                            change_entropy=0.00019552920957721653,
-                            atol_ints=1e-10) # in order to make CI pass
+                            l2=[0.0032743987817220976 0.007246750130235789 0.0],
+                            linf=[0.0027330553477815656 0.0037734958705577526 0.0],
```

Comment:
No, that shouldn't be the reason because the elixir uses `bathymetry_flat`. I am actually not sure why the values changed. I also thought about that briefly, but didn't find a reason why the values should change. The differences are in the order of 1e-8. I thought they are small enough to accept, but I'm not sure. If you have an idea where the difference could come from, I would be interested.
What is also worth mentioning is that the values for the variable bathymetry did not change, compare the [new numbers](https://github.com/JoshuaLampert/DispersiveShallowWater.jl/pull/147/files#diff-9717cb55bc2dcd5ed4467b470a8c9dca30c901e08488cba485dd29f1d18230cbR60) with the [old numbers](https://github.com/JoshuaLampert/DispersiveShallowWater.jl/pull/147/files#diff-7bd90c8f279687056b5c3767e18bd0799a8c78c9b3b8ada1bc10f338221b7228L15).
Another note worth mentioning: I expect the upwind values a few lines further down to change a bit because before the `BBMBBMEquations1D` didn't use the upwind SBP operators, but only the central one.

Reply:
We use error-based step size control. Since the length of the state vector `q` has changed, the (relative) norms are a bit different. Thus, I would expect to get exactly the same results if we used constant time step sizes. Maybe it's worth testing this once to see whether it's purely an effect of the adaptive time integration method?

Reply:
That makes sense. Let me check this quickly.